openfecli/commands/quickrun.py

```diff
@@ -79,9 +89,11 @@ def quickrun(transformation, directory, output):
         if 'inputs' in dd:
             dd['inputs'].pop('settings')
 
-    if output:
-        with open(output, mode='w') as outf:
-            json.dump(out_dict, outf, cls=JSON_HANDLER.encoder)
+    if output is None:
+        output = work_dir / (str(trans.key) + '_results.json')
```

Comment:
I'm not entirely sure on this point. The option to *not* create an output file was left intentionally, on the idea that much of the same info is piped to stdout and that a user might not want to add things to their filesystem.

Reply:
I've occasionally forgot to set `-o` and you drop a lot of info if you don't include it.  It's probably safer just to always write it

Reply:
What if we change the default to write out but keep the option to not write stuff out?